packages/lib-core/src/index.ts

```diff
@@ -8,3 +10,12 @@ export {
 export { useExtensions } from './store/useExtensions';
 export { usePluginInfo } from './store/usePluginInfo';
 export { useResolvedExtensions } from './store/useResolvedExtensions';
+
+// Support APIs, to be consumed by other plugin SDK packages
```

Comment:
@andrewballantyne Proposed to have a separate file to export these support APIs (to keep the main API surface as small as possible), we can do it via follow-up change.

Reply:
We should create a ticket for this.

Reply:
Created: https://issues.redhat.com/browse/HAC-760